core/src/main/kotlin/org/evomaster/core/problem/graphql/service/GraphQLBlackBoxFitness.kt

```diff
@@ -65,4 +64,16 @@ class GraphQLBlackBoxFitness : GraphQLFitness() {
          */
         return null
     }
+
+    override fun getGraphQLError(
```

Comment:
maybe we need a better name... in BB we still have Errors, it is just that we miss the info on the last-executed-line.
Maybe rename with something like `getGraphQLErrorWithLineInfo`

Reply:
Done